oak-store-document/src/test/java/org/apache/jackrabbit/oak/plugins/document/FullGCHelper.java

```diff
@@ -60,6 +63,16 @@ public static void disableFullGCDryRun(final VersionGarbageCollector vgc) throws
         }
     }
 
+    public static VersionGarbageCollector.VersionGCStats gc(final VersionGarbageCollector gc, final long maxRevisionAge, final TimeUnit unit) throws IOException {
+        gc.setFullGcMaxAge(maxRevisionAge, unit); // set full gc max age
+        final VersionGarbageCollector.VersionGCStats stats = gc.gc(maxRevisionAge, unit);
+        if (stats.skippedFullGCDocsCount != 0) {
+            (new Exception("here: " + stats.skippedFullGCDocsCount)).printStackTrace(System.out);
```

Comment:
stdout?

Reply:
this has been added to help in printing the error to console in case of failures while running test cases.